pkg/controller/versions/strategy_v2_6.go

```diff
@@ -461,6 +462,18 @@ func (v *versionStrategyV2_6) Render(ctx context.Context, cr *common.ControllerR
 		return nil, fmt.Errorf("could not set field meshConfig.ingressControllerMode: %v", err)
 	}
 
+	// Enable FIPS automatically if necessary
+	data, err := ioutil.ReadFile("/proc/sys/crypto/fips_enabled")
+	if err != nil {
+		return nil, fmt.Errorf("could not read file /proc/sys/crypto/fips_enabled: %v", err)
+	}
+	if string(data) == "1" {
+		err = spec.Istio.SetField("global.fipsEnabled", true)
```

Comment:
Is it okay to assume that all nodes are fips enabled based on just the node that runs the operator?

Reply:
If I understand correctly [this page](https://docs.openshift.com/container-platform/4.16/installing/installing-fips.html#installing-fips-mode_installing-fips) fips is installed at cluster level, meaning all nodes are either fips-enabled or disabled.